addons/pkg-maven/common/src/main/java/org/commonjava/indy/pkg/maven/content/MavenMetadataGenerator.java

```diff
@@ -483,8 +483,8 @@ public Transfer generateGroupFileContent( final Group group, final List<Artifact
 
         if ( mergingDone )
         {
-            logger.error(
-                    "Merging finished but got some error, which is caused the merging file not created correctly. See merging related error log for details. Merging group: {}, path: {}",
+            logger.info(
+                    "Merging finished but the merging file not created correctly. See merging related log for details. Merging group: {}, path: {}",
```

Comment:
I saw this message in the logs yesterday, and I have to say I didn't immediately see the other logs it was referring to. I wonder if it's expecting that download failures, etc. will have been reported at a lower layer in the stack?

Reply:
I looked at the reason for some such ERRORs and they are just because the requested artifact does not exist in all the member repos of a group. So the result is nothing aggregated. This should be an info and user get 404, which is also right. I see you have fixed something so the 404 is not taken as an error in the response util now.